src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,88 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd: (movie: Movie) => void,
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [valueTitle, setValue] = useState('');
+  const [valueDescription, setValueDescription] = useState('');
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
  const [description, setDescription] = useState('');
```
Set function should be named according to the noun variable. Also, do you really need to duplicate "value" everywhere? I think it's redundant. Please fix all cases